commands_setSellAmount.go

```diff
@@ -63,5 +63,22 @@ func (ssa setSellAmountCmd) ToAuditEvent() types.AuditEvent {
 }
 
 func (ssa setSellAmountCmd) Execute() {
-	consoleLog.Warning("Not implemented: SET_SELL_AMOUNT")
+	autoTxKey := types.AutoTxKey{
+		Stock:  ssa.stock,
+		UserID: ssa.userID,
+		Action: "Sell",
+	}
+	_, found := workATXStore[autoTxKey]
+	if found {
+		// autoTx already exists, we'll need to cancel it.
```

Comment:
`autoTxUtil.go` sounds good.

Could also include your publish function in there, too. Something like `publishAutoTx(type string, key types.AutoTxKey)`.